progress/internal/linear-progress.ts

```diff
@@ -15,7 +15,7 @@ import {Progress} from './progress.js';
  */
 export class LinearProgress extends Progress {
   /**
-   * Buffer amount to display, a fraction between 0 and 1.
+   * Buffer amount to display, a fraction between 0 and max.
```

Comment:
One last suggestion so it matches the change in `progress.md`:

```suggestion
   * Buffer amount to display, a fraction between 0 and `max`.
```

Thank you!